cmd/gardenlet/app/migration.go

```diff
@@ -6,10 +6,150 @@ package app
+// migrateAdminViewerKubeconfigClusterRoleBindings moves the ClusterRoleBindings granting access to the
+// shoot/adminkubeconfig and shoot/viewerkubeconfig subresources from the shoot-core-system managed resource to the
+// shoot-core-gardeneraccess managed resource.
+// TODO(vpnachev): Remove this after v1.132.0 has been released.
+func migrateAdminViewerKubeconfigClusterRoleBindings(ctx context.Context, log logr.Logger, seedClient client.Client) error {
+	namespaceList := &corev1.NamespaceList{}
+	if err := seedClient.List(ctx, namespaceList, client.MatchingLabels(map[string]string{v1beta1constants.GardenRole: v1beta1constants.GardenRoleShoot})); err != nil {
+		return fmt.Errorf("failed listing namespaces: %w", err)
+	}
+
+	var (
+		tasks []flow.TaskFn
+		crbs  = []string{v1beta1constants.ShootProjectAdminsGroupName, v1beta1constants.ShootProjectViewersGroupName, v1beta1constants.ShootSystemAdminsGroupName, v1beta1constants.ShootSystemViewersGroupName}
+	)
+
+	for _, namespace := range namespaceList.Items {
+		if namespace.DeletionTimestamp != nil || namespace.Status.Phase == corev1.NamespaceTerminating {
+			continue
+		}
+
+		tasks = append(tasks, func(ctx context.Context) error {
+			var (
+				shootSystemKey             = client.ObjectKey{Namespace: namespace.Name, Name: shootsystem.ManagedResourceName}
+				shootSystemManagedResource = &resourcesv1alpha1.ManagedResource{}
+
+				gardenerAccessKey             = client.ObjectKey{Namespace: namespace.Name, Name: gardeneraccess.ManagedResourceName}
+				gardenerAccessManagedResource = &resourcesv1alpha1.ManagedResource{}
+			)
+
+			// Get the shoot-core-system managed resource and check if it is already migrated.
+			if err := seedClient.Get(ctx, shootSystemKey, shootSystemManagedResource); err != nil {
+				if apierrors.IsNotFound(err) {
+					log.Info("Managed resource not found, skipping migration", "managedResource", shootSystemKey)
+					return nil
+				}
+				return fmt.Errorf("failed to get ManagedResource %q: %w", shootSystemKey, err)
+			}
+
+			if shootSystemManagedResource.DeletionTimestamp != nil {
+				return nil
+			}
+
+			shootSystemObjects, err := managedresources.GetObjects(ctx, seedClient, shootSystemManagedResource.Namespace, shootSystemManagedResource.Name)
+			if err != nil {
+				if apierrors.IsNotFound(err) {
+					log.Info("Managed resource secret not found, skipping migration", "managedResource", shootSystemKey)
+					return nil
+				}
+				return fmt.Errorf("failed to get objects for ManagedResource %q: %w", shootSystemKey, err)
+			}
+
+			oldShootSystemObjectsCount := len(shootSystemObjects)
+			shootSystemObjects = slices.DeleteFunc(shootSystemObjects, func(obj client.Object) bool {
+				return slices.Contains(crbs, obj.GetName())
+			})
+
+			if oldShootSystemObjectsCount == len(shootSystemObjects) {
+				log.Info("ClusterRoleBindings for shoot/adminkubeconfig and shoot/viewerkubeconfig have already been migrated, skipping", "managedResource", shootSystemKey)
```

Comment:
For consistency with the rest of the `skipping migration` logs:
```suggestion
				log.Info("ClusterRoleBindings for shoot/adminkubeconfig and shoot/viewerkubeconfig have already been migrated, skipping migration", "managedResource", shootSystemKey)
```